### PR TITLE
Initial Just Spaces edits

### PR DESCRIPTION
**Pull request by beamalsky**

### CODE REVIEW

**Comment by beamalsky** [comment]
@reginafcompton, this is ready for your review. I've added notes on what's accomplished here to the README: https://github.com/datamade/django-fobi/tree/just-spaces-edits

Once this is good to go I'll switch the import in the Just Spaces repo to the master branch here, and we can start a better workflow of having PRs structured around single changes!

File: README.rst [hunk 1]
```diff
@@ -6,2463 +6,20 @@ friendly form generator/builder application for Django. With `fobi` you can
...
-
-    ┌──────────────────────────┬───────────────────────┬───────────────────────┐
-    │ `Plugin`                 │ `Users`               │ `Groups`              │
-    ├──────────────────────────┼───────────────────────┼───────────────────────┤
-    │ Text                     │ John Doe              │ Form builder users    │
-    ├──────────────────────────┼───────────────────────┼───────────────────────┤
-    │ Textarea                 │                       │ Form builder users    │
-    ├──────────────────────────┼───────────────────────┼───────────────────────┤
-    │ File                     │ Oscar, John Doe       │ Staff members         │
-    ├──────────────────────────┼───────────────────
```
Comment (reginafcompton) [inline README.rst]:
```suggestion
This fork has been customized to fit the needs of DataMade's
```

**Review by reginafcompton — COMMENTED** [review]
I have not been following the sundry changes you've made to this repo. However, I do know that they behave as expected on the Just Spaces side of the tool. 

I am only going to suggest a simple edit to the README, unless you'd like me to look at any of this code in particular – is there anything you need specific review on, @beamalsky ?

File: README.rst (hunk 1 above)
Comment (reginafcompton) [inline README.rst]:
```suggestion
This fork has been customized to fit the needs of DataMade's
```

**Comment by beamalsky** [comment]
I accepted your README phrasing change, neat new tool! I'm comfortable merging this now if you are.

**Review by reginafcompton — APPROVED** [review]

**Pull request merged**